### ISSUE TITLE
v1.8.2 setup.py is not python2 compatible

### ISSUE THREAD

**Issue by alirizakeles** [confluentinc/confluent-kafka-python#1280]
Description
===========
setup.py of v1.8.2 is not python2 compatible, so pip installation fails with error:

```
Collecting confluent-kafka>=0.9.2
  Downloading https://files.pythonhosted.org/packages/fb/16/d04dded73439266a3dbcd585f1128483dcf509e039bacd93642ac5de97d4/confluent-kafka-1.8.2.tar.gz (104kB)
    100% |████████████████████████████████| 112kB 2.3MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-OAHgZ9/confluent-kafka/setup.py", line 12, in <module>
        with open(os.path.join(work_dir, 'README.md'), encoding='utf-8') as f:
    TypeError: 'encoding' is an invalid keyword argument for this function

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-OAHgZ9/confluent-kafka/
```
I think this line causes the problem:
https://github.com/confluentinc/confluent-kafka-python/blob/v1.8.2/setup.py#L12

How to reproduce
================
On a python2 environment, try installing via pip.

Checklist
=========
Please provide the following information:

 - [x] confluent-kafka-python: v1.8.2
 - [x] Operating system: ubuntu 16.04, python: 2.7.12
 - [x] Critical issue

**Comment by edenhill** [comment]
Will be fixed in the upcoming v1.9.0 release.
But you should really start upgrading to Python 3.

**Comment by OneCricketeer** [comment]
Pip itself dropped support for Python2, by the way
